alerts/cluster-monitoring-operator/TargetDown.md

```diff
@@ -0,0 +1,269 @@
+in Openshift document for more details.
+
+### Target Resource Exhaustion
+
+Check if the target pod resource utilization is too high, causing it to become unresponsive.
+To view usage of CPU and memory, we can check these metrics:
+- CPU Usage
+  * `container_cpu_usage_seconds_total`: Cumulative CPU usage in seconds.
+  * `pod:container_cpu_usage:sum` Represents the average CPU load of a pod over
+  the last 5 minutes.
+  * `container_cpu_cfs_throttled_seconds_total` Duration when the CPU was throttled
+  due to limits.
+- Memory Usage
+  * `container_memory_usage_bytes` Current memory usage in bytes.
+  * `container_memory_rss` Resident set size in bytes. This metric gives an
+  understanding of the memory actively used by a pod.
+  * `container_memory_swap` Swap memory usage in bytes.
+  * `container_memory_working_set_bytes` Total memory in use. This includes all
+  memory regardless of when it was accessed.
+  * `container_memory_failcnt` Cumulative count of memory allocation failures.
+- File System Usage
+  * `container_fs_reads_bytes_total` and `container_fs_writes_bytes_total`
+  Cumulative filesystem read and write operations in bytes.
+  * `container_fs_reads_total` and `container_fs_writes_total` Cumulative
+  filesystem read and write operations in bytes.
+  * `kubelet_volume_stats_available_bytes` The free space in a volume in bytes.
+
+If some basic metrics are not available, we can also use this command to get
+ CPU and memory usage of a pod:
+```bash
+oc top pod $POD_NAME
+```
+As well as this command for volume free space:
+```bash
+oc exec -n $NAMESPACE $POD_NAME -- df
+```
+
+### Target Application or Service Failure
+
+Investigate if the application or service running on the target is experiencing
+issues or has even crashed. Review logs and last metrics from the target pod to
+identify any errors or crashes.
+
+Here is [a guide to investigate pod issues](https://docs.openshift.com/container-platform/4.13/support/troubleshooting/investigating-pod-issues.html)
+in Openshift.
+
+### Incorrect Target Configuration
+
+Verify that the scrape target configuration in Prometheus is correct. This
+configuration is generated from the `ServiceMonitor` and `PodMonitors`.
+We can get the `ServiceMonitor` and `PodMonitors` related to the alert with this
+command using the `namespace` and `job` label:
+```bash
+oc get servicemonitor -n $NAMESPACE -o yaml $JOB
+```
+Check the target's port, selector, scheme, TLS settings, etc for invalid values.
+Please refer to [the Prometheus Operator API document](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md)
+for detailed specification of `ServiceMonitor` and `PodMonitors`.
+
+About the TLS settings, even if the values are correct, it may still fail due to
+certificate expiration. Please refer to the next section for details.
+
+### Expired SSL/TLS Certificates
```

Comment:
I will add a phrase telling user to contact openshift support when automatic renewal does not happen. but keep the steps to force a renewal in the runbook in case the user want an immediate resolution of the issue.